docs/_redirects

```diff
@@ -9,7 +9,7 @@
 /guide/index.html	/guides/
 /testing/testing-helpers.html	/docs/testing/helpers/
 /demoing/	/docs/demoing/storybook/
-/developing/best-practices.html	/guides/knowledge/lit-element/lifecycle/
+/developing/best-practices.html	/guides/knowledge/lit/lifecycle/
```

Comment:
Do we need to add _new_ redirects to get the previous redirect locations to forward as well?

Reply:
as in
```suggestion
/guides/knowledge/lit-element/lifecycle/ /guides/knowledge/lit/lifecycle/
```
?

I don't think so as long as nothing is pointing to `/guides/knowledge/lit-element`, a quick search doesn't show me any at least. When doing this change I basically looked for `lit-` as a search filter, and edited everything I found essentially except for blogs, guides related to open-wc/create (still using old lit) and webcomponents.dev code samples, so I think it should be ok.